pkg/overlay/cache.go

```diff
@@ -176,7 +166,7 @@ func (cache *Cache) IsNew(node *NodeDossier) bool {
 
 // IsOnline checks if a node is 'online' based on the collected statistics.
 func (cache *Cache) IsOnline(node *NodeDossier) bool {
-	return time.Now().Sub(node.Reputation.LastContactSuccess) < OnlineWindow &&
+	return time.Now().Sub(node.Reputation.LastContactSuccess) < cache.preferences.OnlineWindow &&
```

Comment:
optional/nit: just because I feel like this is a long check, it could maybe use a comment to explain the values it's using in case the constraints ever change.

Reply:
I'm not sure I understand.  Are you saying just put a comment in that tell you what the default is?